### PR TITLE
Enable awsecscontainermetrics receiver, and metricstransform processors

### PR DESCRIPTION
**Pull request by hossain-rayhan**
This change will enable the following components.

Receiver:
`awsecscontainermetrics`

Processors:
`metrictransform processors`

### CODE REVIEW

**Comment by mxiamxia** [comment]
Hi Rayhan, please attach the integration test result for awsecscontainermetrics receiver in this PR and we can merge it into main branch. thank you for the help.

**Comment by wyTrivail** [comment]
We still have the empty dimension issue today, what's our plan to fix it? I feel this is a blocker to merge this PR, correct me if I'm wrong. Thanks

**Comment by wyTrivail** [comment]
any documentation?

**Comment by hossain-rayhan** [comment]
@wyTrivail , I believe we had a discussion last Friday, and I updated our integration test PR description and comment. Check it here- https://github.com/aws-observability/aws-otel-test-framework/pull/40 

Dimension setting is a different work, and we are investigating on that. 

For documentation- check the updated [README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/receiver/awsecscontainermetricsreceiver/README.md).

**Comment by hossain-rayhan** [comment]
> Hi Rayhan, please attach the integration test result for awsecscontainermetrics receiver in this PR and we can merge it into main branch. thank you for the help.

Screen Shot 2020-10-26 at 4 20 48 PM

**Review by mxiamxia — APPROVED** [review]

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -455,6 +439,9 @@ jobs:
       - name: Run testing suite on eks
```
Comment (mxiamxia) [inline .github/workflows/CI.yml:439]: typo on `Run testing suite on eks` here?

**Pull request merged**